src/js/common/schemaform/save-load-actions.js

```diff
@@ -135,29 +139,33 @@ export function saveInProgressForm(formId, version, returnUrl, formData) {
       body
     }).then((res) => {
       if (res.ok) {
-        dispatch(setSaveFormStatus(SAVE_STATUSES.success));
-      } else {
+        // TODO update this when the backend api is updated to return the timestamp
+        // return res.json();
+        return Promise.resolve({ timestamp: moment().valueOf() });
+      }
+
+      return Promise.reject(res);
+    })
+    .then(({ timestamp }) => {
+      dispatch(setSaveFormStatus(SAVE_STATUSES.success, timestamp));
+    })
+    .catch((resOrError) => {
+      if (resOrError instanceof Response) {
         // TODO: If they've sat on the page long enough for their token to expire
         //  and try to save, tell them their session expired and they need to log
         //  back in and try again. Unfortunately, this means they'll lose all
         //  their information.
-        if (res.status === 401) {
+        if (resOrError.status === 401) {
           dispatch(setSaveFormStatus(SAVE_STATUSES.noAuth));
         } else {
           dispatch(setSaveFormStatus(SAVE_STATUSES.failure));
         }
-      }
-      return Promise.resolve(); // For unit testing
-    }).catch((error) => {
-      // Probably a network error has occurred
-      // TODO: Log this in GA or something
-      console.error('Error saving form:', error.message); // eslint-disable-line no-console
-      dispatch(setSaveFormStatus(SAVE_STATUSES.failure));
-      if (__BUILDTYPE__ === 'development') {
-        return Promise.reject(); // For unit testing
```

Comment:
If this isn't in here, some of the unit tests won't actually run properly. You won't get any failures, but the `expect()` statements in the `.catch()`es won't be run (unless I'm missing something with the changes you made in the unit tests).